content/en/tutorials/upgrading_eks.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Upgrading Kubernetes Version on AWS"
```

Comment:
we will also need some upgrade notes for helm charts that are not specific for the cloud provider, what is the plan for these?
it seems that all the k8s upgrade documentation could be regrouped.

Reply:
We _hope_ that it remains provider agnostic, but I highly doubt that it will remain like that. For instance, right now we rely on EKS to handle EBS volume creation for persistent storage, but they may force us to use their EBS CSI driver solution in the future